From 5fd26bf7da4f5c5aa97e86ef2dc4f4c191dc6fdb Mon Sep 17 00:00:00 2001
From: Dave Eddy <dave@daveeddy.com>
Date: Mon, 19 Mar 2018 15:39:54 -0400
Subject: [PATCH] TRITON-240 zoneevent should be called with an identifier
 string Reviewed by: Cody Peter Mello <cody.mello@joyent.com> Reviewed by:
 Josh Wilsdon <josh@wilsdon.ca> Approved by: Josh Wilsdon <josh@wilsdon.ca>

---
 lib/vm.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/vm.js b/lib/vm.js
index 5715c40..2f7bbf7 100644
--- a/lib/vm.js
+++ b/lib/vm.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 /*
@@ -204,7 +204,7 @@ EventWatcher.prototype.startZoneWatcher = function () {
 
     var watcher = this.zoneeventWatcher = spawn(
         '/usr/vm/sbin/zoneevent',
-        [],
+        ['-i', 'net-agent'],
         {'customFds': [-1, -1, -1]});
 
     log.info('zoneevent running with pid ' + watcher.pid);
-- 
2.21.0

